OracleWebCenterSites/README.md

```diff
@@ -94,20 +94,26 @@ Sign in to [Oracle Container Registry](https://container-registry.oracle.com). C
 ### C. To download Oracle WebCenter Sites Docker files and binary file.
 
 1. Clone the GitHub repository. URLs for cloning: 
-
  - HTTPS:[ https://github.com/oracle/docker-images.git](https://github.com/oracle/docker-images.git) 
-
  - SSH: [git@github.com:oracle/docker-images.git](git@github.com:oracle/docker-images.git).
-
- The repository contains Docker files and samples to build Docker images for Oracle products and open source projects.
-
+    
+    The repository contains Docker files and samples to build Docker images for Oracle products and open source projects.
 2. Download Oracle WebCenter Sites 12c R2 12.2.1.3 binary from [Oracle Technology Network](http://www.oracle.com/technetwork/middleware/webcenter/sites/downloads/index.html).
```

Comment:
I would specify the exact filename or option as there are a lot of options on this web page.